docs/docs/behaviors/soft-off.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Soft Off Behavior
+sidebar_label: Soft Off
+---
+
+## Summary
+
+The soft off behavior is used to force the keyboard into an off state. Depending on the specific keyboard hardware, the keyboard can be turned back on again either with a dedicated on/off button that is available, or using the reset button found on the device.
+
+For more information, see the [Soft Off Feature](../features/soft-off.md) page.
```

Comment:
I think we might want to note somewhere that `CONFIG_ZMK_PM_SOFT_OFF=y` is required for this behavior to work.

Reply:
Pinging @petejohanson on this, in case you missed it.

Reply:
Add a link to the actual config section where it's documented, rather than duplicating it here. Thoughts?

Reply:
Sure, it seemed simple to note to me because it needed a single flag but this is fine too.